static/js/modules/download.js

```diff
@@ -137,7 +137,7 @@ DownloadItem.prototype.refresh = function() {
     contentType: 'application/json',
     dataType: 'json'
   });
-  this.promise.then(this.handleSuccess.bind(this));
+  this.promise.done(this.handleSuccess.bind(this));
```

Comment:
Interesting, this required a change to `done`?  In reading the guide I would've thought `then` would still be preferred as that is the new way forward with the `Deferred` object changes.  It appears we're still using the old style, though, [according to this part](https://jquery.com/upgrade-guide/3.0/#callback-exit), which would make sense given this change.